.eslintrc.js

```diff
@@ -42,7 +42,7 @@ module.exports = {
     "eol-last": "warn",
     eqeqeq: ["error", "smart"],
     "no-case-declarations": "off",
-    "no-console": ["error", { allow: ["warn", "error"] }],
+    // "no-console": ["error", { allow: ["warn", "error"] }],
```

Comment:
I think this change must've gotten checked in by accident, right?

Reply:
I'll second this.  We want to leave this lint setting on.